spec/requests/carto/api/overlays_controller_spec.rb

```diff
@@ -59,6 +61,48 @@
       end
     end
 
+    it 'lists all overlays with advanced ui user' do
+      begin
+        orig_host = host
+        # Enable the bbg_pro_ui flag to have all 5 layers return
+        ::User.any_instance.stubs(:has_feature_flag?).with('bbg_pro_ui').returns(true)
```

Comment:
Try to avoid using the `any_instance` method. You have a specific user object below, stub that one. This ensures that the stubs are tied to this spec, and saves the need to do a generic unstub afterwards.